pages/myinfo/index.page.tsx

```diff
@@ -49,6 +57,7 @@ export default function MyInfo({ children }: { children: ReactNode }) {
             <Link
               key={index}
               href={link.href}
+              onClick={link.handleOnClick ? link.handleOnClick : undefined}
```

Comment:
요렇게 단축해서 사용해도 괜찮을지 제안해봅니당!
```suggestion
              onClick={link.handleOnClick || undefined}
```

Reply:
좋아용!!